### PR TITLE
Feature/refactor: Divided Documentation part into smaller parts

### PR DESCRIPTION
**Pull request by UlyanaHashchuk**
## Description
Added new folders in Documentation:

* HowItWorks
* ApiEndpoints
* ComingSoon 

Above components included in main `index.tsx` in Documentation.

### CODE REVIEW

**Comment by norbertdurcansk** [comment]
Tip: 
* Start title with UpperCase
Write description with power of [Markdown](https://guides.github.com/features/mastering-markdown/)
* Assign reviewer in Github -> on the right

## Description

- Added new folders in Documentation:
   - HowItWorks
   -  ApiEndpoints
   -  ComingSoon with code components included in main `index.tsx` in Documentation.

File: modules/Documentation/components/ApiEndpoints/index.tsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import Text from '../../../../components/Text';
+import Button from '../../../../components/Button';
+import { Container, Code } from '../../index.styled';
+import Snippet from '../Snippet';
+
+export default ({GRAPHQL_SERVER_URI, clientId, clientSecret}) => {
+  return (
```
Comment (norbertdurcansk) [inline modules/Documentation/components/ApiEndpoints/index.tsx]: Would use without `return` and `{` 

()=>(<Container/>)

File: modules/Documentation/components/ComingSoon/index.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import Text from '../../../../components/Text';
+
+export default () => {
+  return (
```
Comment (norbertdurcansk) [inline modules/Documentation/components/ComingSoon/index.tsx]: Same like above

File: modules/Documentation/components/HowItWorks/index.tsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import Text from '../../../../components/Text';
+import Button from '../../../../components/Button';
+import { Container, Code } from '../../index.styled';
+import Snippet from '../Snippet';
+
+export default ( {ILP_SERVER_URL, clientId} ) => {
```
Comment (norbertdurcansk) [inline modules/Documentation/components/HowItWorks/index.tsx]: Same like above

Also consider adding `eslint` to your IDE for automatic code formatting

File: modules/Documentation/components/HowItWorks/index.tsx [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+  return (
+    <Container>
```
Comment (norbertdurcansk) [inline modules/Documentation/components/HowItWorks/index.tsx]: Don't use `Container`. It was not there. U can just use empty tag 

```
<>
...elements

</>
```

File: package.json [hunk 5]
```diff
@@ -34,7 +34,8 @@
     "styled-components": "^5.1.0",
     "styled-system": "^5.1.5",
     "subscriptions-transport-ws": "^0.9.16",
-    "ws": "^7.3.0"
+    "ws": "^7.3.0",
+    "yarn": "^1.22.4"
```
Comment (norbertdurcansk) [inline package.json]: Remove `yarn` from dependecies. Yarn should be global not in project

**Review by norbertdurcansk — CHANGES_REQUESTED** [review]
Added few comments. :) Good job 💸

**Comment by UlyanaDurcanska** [comment]
Pushed changes :sweat_smile:

**Review by norbertdurcansk — APPROVED** [review]

**Pull request merged**